states/pools.sls

```diff
@@ -39,10 +39,10 @@ libvirt_virsh_pool_{{ name }}:
     - run
   {% if p.ensure|default('running') in ['present', 'running'] %}
     - name: virsh pool-define {{ po_path }}
-    - unless: virsh -q pool-list --all | grep -q '^{{ name }}'
+    - unless: virsh -q pool-list --all | grep -Eq '^\s?{{ name }}'
```

Comment:
please replace ``\s+`` by ``\s*``, to make backwards-compatiblity working (in case there's no space between ^ and ``{{ name }}``)